styles.css

```diff
@@ -357,7 +398,17 @@ body {
 }
 
 #directorHeading {
-color: #cccccc
+  color: #cccccc
+}
+
+.servicesIconImages {
+  height: 3.5rem;
+  width: 3.5rem;
+  text-align: center;
```

Comment:
it would be better if we directly apply the bootstrap css `text-center` class to the `servicesIconImages` element instead of defining it here (even though it will be done multiple times since you have a couple of`servicesIconImages` classes defined.

reason being it would be easier to know that this styling of `text-align: center` is applied to the element by read the class name without having to go into the css file and find out through `servicesIconImages`

Reply:
I used `mx-auto d-block` instead of `text-center` cuz for some reason `text-center` doesn't work on an image element, I'm guessing?

The only other occasion I specified `text-align: center;` is on line 52 for `#directorProfilePictureDiv`.. because it's specific to the media query.

Reply:
@samuelkhew  yea you can't apply `text-center` to an `img` element..you can only apply it to the `div` that wraps the `img` element..but your approach is fine as well